plugins/generator-1.14.4/forge-1.14.4/procedures/spawn_entity.java.ftl

```diff
@@ -7,9 +7,13 @@
 			Entity entityToSpawn = new ${entity}(${entity.toString().replace(".CustomEntity", "")}.entity, world.getWorld());
 		</#if>
 		entityToSpawn.setLocationAndAngles(${input$x}, ${input$y}, ${input$z}, world.getRandom().nextFloat() * 360F, 0);
-
+		<#if !entity.toString().contains(".CustomEntity")>
+		    ((${generator.map(field$entity, "entities", 0)}) entityToSpawn).setGrowingAge(-24000);

```

Comment:
Wouldn't it be better to allow users to set the growing time by themselves? Applies to all occurrences of similar code.

Reply:
Done!